src/Application/Application.ts

```diff
@@ -212,7 +212,7 @@ export class Application {
       this.spinner.maybeSucceed();
       this.spinner.maybeCreateMessageForSpinner('Checking for results (this could take a minute)');
       logMessage('Polling Nexus IQ Server for report results', DEBUG, resultUrl);
-      requestService.asyncPollForResults(`${args.server}/${resultUrl}`, (x) => {
+      requestService.asyncPollForResults(`${requestService.host}/${resultUrl}`, (x) => {
```

Comment:
This is a great catch, but we probably should just pass in the `resultURL` and then do the append of the host inside the RequestService, to likely avoid some stupid bug that Jeffry created like this in the near future :)

Reply:
tested it and it works now, will merge since its breaking and we can look at the stylistic stuff in the beta

Reply:
@ajurgenson55 Sounds good! I was investigating how to best implement @DarthHater's request, i'll open it as an issue to fix on another branch.